ansible/roles/hcv-haproxy-configure/files/up.map.template

```diff
@@ -0,0 +1 @@
+up true
```

Comment:
I think this doesn't belong as a template

Reply:
agreed, that was a placeholder and it's now deleted. I do think it may make sense to make it a template in the future.